src/render/jsapi.js

```diff
@@ -0,0 +1,38 @@
+'use strict'
+
+const { L, S, Joi, iterateKeysToArstr, C } = require('../utils')
+
+module.exports = (models, config) => {
+  const routes = iterateKeysToArstr(models, (modelName, model) => {
+    const route = (name, handler) => {
+      return `async ${name} ${handler},`
+    }
+
+    const getRoute = modelName => {
+      return route(`get${C(modelName)}`, `(id) {
+      const obj = await DBM.db.getById('${modelName}', id)
+      return obj
+    }`)
+    }
+
+    const setRoute = modelName => {
+      return route(`set${C(modelName)}`, `(id, val, userId = 0) { // using user 0 since we are the server
```

Comment:
not sure if using NULL/unset is better as this is somehow special from 0 which is also a number, but this could cause trouble with the SQL based schemas

Reply:
Yeah I wasn't sure how to handle the internal api requests if we have logging enabled, for now I just wanted a way to distinguish http-api from js-api calls for debugging.
We should probably add an option so the user can decide what the internal id should be.